tools/RAiDER/gnss/processDelayFiles.py

```diff
@@ -143,12 +162,20 @@ def create_parser():
         default=os.getcwd()
     )
 
+    p.add_argument(
+        '--gnss', dest='gnss_file',
+        help=dedent("""\
+            Optional .csv file containing GPS Zenith Delays. Should contain columns "ID", "ZTD", and "Datetime"
+            """),
+        default=None
+    )
+
```

Comment:
Can you add run file examples to the helper?

Reply:
@dbekaert done.

Reply:
There was already an example command. I added another to cover the case of no gnss files.